src/main/java/com/online/booking/service/VehicleService.java

```diff
@@ -1,7 +1,15 @@
 package com.online.booking.service;
 
+import com.online.booking.models.VehicleModel;
 import org.springframework.stereotype.Service;
 
+import java.util.List;
+
 @Service
 public interface VehicleService {
+    void createVehicle(VehicleModel vehicle);
+    void editVehicleByID(int vehicleID);
+    void deleteVehicleByID(int vehicleID);
```

Comment:
if return type is void, how would calling function know that deleting was successful or not ?